File: campus-accessibility-frontend/src/types/complaintStatus.ts
//Οι καταστάσεις των παραπόνων
export type ComplaintStatus =
    | "New"
    | "UnderReview"
    | "InProgress"
    | "Pending"
    | "Resolved"
    | "Rejected"

export const COMPLAINT_STATUS_LABELS: Record<ComplaintStatus, string> = {
    New: "Νέα",
    UnderReview: "Υπό εξέταση",
    InProgress: "Σε εξέλιξη",
    Pending: "Εκκρεμεί",
    Resolved: "Επιλύθηκε",
    Rejected: "Απορρίφθηκε",
}

File: CampusAccessibilityBackend/CampusAccessibilityBackend/Enums/ComplaintStatus.cs
namespace CampusAccessibilityBackend.Enums
{
    public enum ComplaintStatus
    {
        New,
        UnderReview,   //ο admin το έχει δει και διερευνά το πρόβλημα/μιλάει με το τμήμα
        InProgress,   // η τεχνική υπηρεσία ασχολείται ενεργά με το πρόβλημα
        Pending,       //Δεν υπάρχει ακόμη απόκριση από την τεχνική υπηρεσία ή όποιο άλλο τρίτο φορέα
        Resolved,     
        Rejected      //δεν μπορεί να επιλυθεί/ εκτός αρμοδιότητας
    }
}